lab-ron/server.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
This looks great, but doesn't run because of the error in your client-pool.js file, please see my comments above for more information.